bin/kyuubi

```diff
@@ -48,6 +49,7 @@ function kyuubi_rotate_log() {
     num=${KYUUBI_MAX_LOG_FILES}
   else
     echo "Error: KYUUBI_MAX_LOG_FILES must be a positive number, but got ${KYUUBI_MAX_LOG_FILES}"
+    # shellcheck disable=SC2242
     exit -1
```

Comment:
this can be fixed in place, we just need to return a no-zero value here, 1 should be good.

Reply:
actually I try this in my fork project, the shell script has many grammar mistake, so I use #shellcheck jump that, I want fix these bug in issue #4057 before do this shellcheck

Reply:
what do you think about this idea

Reply:
> the shell script has many grammar mistake

the shell script grammar is flexible; some are style issues, and should not be treated as bug/mistake, because they have worked as expected.

> ... fix these bug in issue https://github.com/apache/kyuubi/issues/4057 before do this shellcheck

w/o shellcheck, how to verify the fix?

Reply:
en, I think I can fix some grammar mistake, then other problems we can use # to ignore them

Reply:
sgtm